src/rp2_common/pico_runtime/runtime.c

```diff
@@ -276,6 +276,7 @@ void __attribute__((noreturn)) panic_unsupported(void) {
     panic("not supported");
 }
 
+#ifndef PICO_PANIC_NO_STRINGS_ON_TARGET
 // PICO_CONFIG: PICO_PANIC_FUNCTION, Name of a function to use in place of the stock panic function or empty string to simply breakpoint on panic, group=pico_runtime
 // note the default is not "panic" it is undefined
 #ifdef PICO_PANIC_FUNCTION
```

Comment:
Did you consider defining PICO_PANIC_FUNCTION?

Reply:
Should I doxygen it here or it is more platform.h place?

Reply:
I've misunderstood see https://github.com/raspberrypi/pico-sdk/issues/1770#issuecomment-2245879949 for benchmarks